src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -1244,11 +1244,8 @@ variable *operating_system* (task scope) resolves to variable *operating_system_
 ----
 
 === Fork environment
-A fork execution environment is a new Java Virtual Machine (JVM) which is started exclusively to execute a task. This task may need to use some third-party libraries, so it can be executed properly. To do that, the list of path elements to be added when starting a new JVM can be specified inside the `Additional Classpath` field.
+A fork execution environment is a new forked Java Virtual Machine (JVM) which is started exclusively to execute a task. This task may need to use some third-party libraries, so it can be executed properly. To do that, the list of path elements to be added when starting a new JVM can be specified inside the `Additional Classpath` field.
```

Comment:
For example, this task may ...  To do that-> In this example, the list of path elements to be added when starting a new JVM will be specified inside the `Additional Classpath` field.